creating_images/guidelines.adoc

```diff
@@ -268,11 +268,56 @@ written to be a particular user ID, and then runs the image using that same user
 ID. This ensures the volume is only accessible to the appropriate container, but
 requires the image be able to run as an arbitrary user ID.
 
+Running a container with an arbitrary uid also has the benefit of ensuring that a 
```

Comment:
You should either change uid to 'user ID' or change the instances of 'user ID' in the previous paragraph to 'uid'